pkg/controllers/openshift-route-controller-manager.go

```diff
@@ -46,7 +52,7 @@ const (
 
 func NewRouteControllerManager(cfg *config.Config) *OCPRouteControllerManager {
 	s := &OCPRouteControllerManager{}
-	s.configure(cfg)
+	s.configErr = s.configure(cfg)
```

Comment:
It looks like maybe we should change the API of the constructors to allow returning errors. I'm surprised none of the others have errors they might return. We'll deal with that separately from this PR.

Reply:
I have coppied this concept from cluster-policy-controller so there are at least two places  :) . +1 for changing the API for this